tsec-http4s/src/main/scala/tsec/authentication/package.scala

```diff
@@ -79,12 +96,12 @@ package object authentication {
       * [[org.http4s.Response.notFound]], which generates a 404, for any request
       * where `pf` is not defined.
       */
-    def apply[A, I, F[_]](
+    def apply[I, A, F[_]](
```

Comment:
this change was because of partial unification bug ?

Reply:
yes.

Reply:
:+1:

Reply:
Also to be clear: `A, I, F[_]` was changed because of the partial unification bug, then the swap to `I, A, F` was because the result type is `I, A, F`. It felt weird to hvae the type params in the apply method in different order of the result type.